pycalphad/core/custom_autowrap.py

```diff
@@ -151,6 +160,9 @@ def _get_loop_opening_ending(self, indices):
             close_lines.append("}")
         return open_lines, close_lines
 
+    _print_ComplexInfinity = CCodePrinter._print_Infinity
```

Comment:
Do we need a `_print_NegativeComplexInfinity = CCodePrinter._print_NegativeInfinity`?

Reply:
I don't think so, but it couldn't hurt. This code is necessary because SymPy differentiation can sometimes produce complex infinity symbols. I think if we were consistent in enforcing assumptions those piecewise paths wouldn't be generated, but ignoring the imaginary part seems to work in this application.